### PR TITLE
unite 4 action server to 1

### PR DESCRIPTION
**Pull request by yuokamoto**

### CODE REVIEW

File: grippers/rr_suction_pump/launch/pisco_svb18.launch [hunk 1]
```diff
@@ -1,13 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <launch>
 	<rosparam file="$(find rr_suction_pump)/config/pisco_svb18.yaml" />
```
Comment (mlautman) [inline grippers/rr_suction_pump/launch/pisco_svb18.launch]: Convert tabs to spaces in sublime.

File: grippers/rr_suction_pump/src/suction_pump_action_server.cpp [hunk 2]
```diff
@@ -21,115 +21,143 @@ class SuctionPumpActionServer
...
         nh.getParam("/output_normally_on", outputNormallyOn);
         nh.getParam("/input_normally_on", inputNormallyOn);
-        for (int i = 0; i < trigger_num; i++) {
-            nh.getParam("trigger_gpio" + std::to_string(i), trigger);
-            trigger_gpio.push_back(trigger);
-        }
-        for (int i = 0; i < status_num; i++) {
-            nh.getParam("status_gpio" + std::to_string(i), status);
-            status_gpio.push_back(status);
+        std::cout << _pump_num << ' ' << outputNormallyOn << ' ' << inputNormallyOn << std::endl;
```
Comment (mlautman) [inline grippers/rr_suction_pump/src/suction_pump_action_server.cpp]: don't use cout when you could alternatively use rosconsole

File: grippers/rr_suction_pump/src/suction_pump_action_server.cpp [hunk 3]
```diff
@@ -21,115 +21,143 @@ class SuctionPumpActionServer
...
     }
 
     bool init()
-    {
-        if (_pump->init(_config)) {
-            _server.start();
-            _pump->set(false); // set off
-            ROS_INFO("%s: Started", _action_name.c_str());
-            return true;
+    {
```
Comment (mlautman) [inline grippers/rr_suction_pump/src/suction_pump_action_server.cpp]: You should be configuring your editor to automatically trim whitespace

File: grippers/rr_suction_pump/src/suction_pump_action_server.cpp [hunk 4]
```diff
@@ -21,115 +21,143 @@ class SuctionPumpActionServer
...
 
     bool init()
-    {
-        if (_pump->init(_config)) {
-            _server.start();
-            _pump->set(false); // set off
-            ROS_INFO("%s: Started", _action_name.c_str());
-            return true;
+    {   
+        for (int i = 0; i < _pump_num; i++) {
```
Comment (mlautman) [inline grippers/rr_suction_pump/src/suction_pump_action_server.cpp]: add clang to this project asap

File: grippers/rr_suction_pump/src/suction_pump_action_server.cpp [hunk 5]
```diff
@@ -21,115 +21,143 @@ class SuctionPumpActionServer
...
+            if (!_pumps[i]->init(_config)) {
+                return false;
+            }
+            _pumps[i]->set(false); // set off
         }
-        return false;
+        _server.start();
+        ROS_INFO("%s: Started", _action_name.c_str());
```
Comment (mlautman) [inline grippers/rr_suction_pump/src/suction_pump_action_server.cpp]:
```suggestion
        ROS_INFO_NAMED(_action_name, "Started");
```

File: grippers/rr_suction_pump/src/suction_pump_action_server.cpp (hunk 4 above)
Comment (yuokamoto) [reply grippers/rr_suction_pump/src/suction_pump_action_server.cpp]: already there
https://github.com/rapyuta-robotics/rr_grippers_ros/blob/master/format_cpp.sh

**Comment by athish-t** [comment]
@yuokamoto -san, are we waiting for @bhuvanchandra 's approval, or shall we merge this? This is needed asap to run the robot with reliable gripping

**Pull request merged**